### ISSUE TITLE
Cutting is not undoable (if first action)

### ISSUE THREAD

**Issue by uli-on** [maelchiotti/LocalMaterialNotes#239]
Cutting text via the contextual menu can not be reverted by the Undo button. 
EDIT Above is only valid if performed as the first step after opening the note.
MN v1.7.2
Android 13
Fairphone FP3

**Comment by maelchiotti** [comment]
Indeed. I'll report this issue to the editor library's maintainers. But since it's a very minor issue, it'll probably have a low priority.
